status/versions.go

```diff
@@ -259,3 +274,27 @@ func kubernetesVersion() (*ExternalServiceInfo, error) {
 	}
 	return nil, err
 }
+
+func canIGetIstioPods(authToken string) (bool, error) {
+	c := config.Get()
+
+	client, err := clientFactory.GetClient(authToken)
+	if err != nil {
+		return false, err
+	}
+
+	ssars, err := client.GetSelfSubjectAccessReview(c.IstioNamespace, "", "pods", []string{"list"})
```

Comment:
I'm not sure it's a subjectaccessreview that we need here. I would rather contact the related services (grafana/jaeger) to make sure access is not forbidden.

Subject access review is using k8s API to get permissions on k8s resources, e.g. pods (as being done here); but does the user really have to get pods list permission to access Grafana? Seems to me that it's unrelated.

Reply:
cc @mwringe

Reply:
The subject access review is because the oauth server has been configured using that.

I don't think there is a problem is we also just check if we can access it or not. That might actually be better since its makes configuration easier.